bitmovin-analytics-datasource/src/types.ts

```diff
@@ -18,3 +18,18 @@ export interface MyDataSourceOptions extends DataSourceJsonData {
   tenantOrgId?: string;
   adAnalytics?: boolean;
 }
+
+export enum QUERY_INTERVAL {
+  SECOND = 'SECOND',
```

Comment:
do we support `SECOND`, `MONTH` and `AUTO` in ARS/AQS ?

Reply:
`AUTO` is a type that we will show just in the grafana interval dropdown, once the query editor is implemented. When customer chooses `AUTO`  interval the plugin will automatically calculate the right interval and then send the corresponding query. 

In ARS we do support `SECOND` and `MONTH` Interval. But not sure how much sense a `SECOND` Interval in general makes and as far as I understood compressed data is only available through the dashboard so  I think here it does not make much sense. I will raise it with the team though.

Reply:
I'll delete the `AUTO` interval for this PR and then add it for the QueryEditor PR where I actually need the UI placeholder.

Also deleting the `SECONDS` and `MONTH` interval as discussed in standup.